pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = [ "setuptools", "setuptools_scm" ]
+build-backend = "setuptools.build_meta"
+ 
+[project]
+name = "ts-fbs-utils"
+description = "Utility package for the Feature Based Scheduler."
+license = { text = "GPL" }
+classifiers = [ "Programming Language :: Python :: 3" ]
+urls = { documentation = "https://ts-scheduler.lsst.io", repository = "https://github.com/lsst-ts/ts_fbs_utils" }
+dynamic = [ "version" ]
+ 
+[tools.setuptools]
+package-data = {"" = "*.csv"}
+ 
+[tool.setuptools.dynamic]
+version = { attr = "setuptools_scm.get_version" }
+ 
+[tool.setuptools.packages.find]
+where = [ "python" ]
+ 
+[tool.setuptools_scm]
+write_to = "python/lsst/ts/fbs/utils/version.py"
+write_to_template = """
+# Generated by setuptools_scm
+__all__ = ["__version__"]
+__version__ = "{version}"
+"""
+ 
+[tool.pytest.ini_options]
+addopts = "--black --flake8 --mypy --ignore-glob=*/version.py"
+flake8-ignore = ["E133", "E203", "E226", "E228", "N802", "N803", "N806", "N812", "N813", "N815", "N816", "W503", "doc/conf.py ALL", "__init__.py ALL", "version.py ALL", "tests/.tests ALL"]
```

Comment:
We decided to drop everything after W503 because it basically isn't necessary. Up to you if you leave it there.